packages/astro/src/core/errors/overlay.ts

```diff
@@ -636,7 +636,8 @@ class ErrorOverlay extends HTMLElement {
 			const codeContent = code.querySelector<HTMLDivElement>('#code-content');
 
 			if (codeHeader) {
-				const cleanFile = err.loc.file.split('/').slice(-2).join('/');
+				const separator = err.loc.file.includes('/') ? '/' : '\\';
```

Comment:
Overlay didn't account for Windows separators here which and this change fixes it.